src/cljs/nr/gameboard/board.cljs

```diff
@@ -1712,7 +1712,8 @@
                 @encounters)
             [run-div side run encounters]
             :else
-            [basic-actions button-pane-args])])})))
+            (when false [basic-actions button-pane-args]))])})))
```

Comment:
this can't possibly be the right condition lol

Reply:
Definitely not lol, supposed to be no condition there